diff --git a/node_modules/@cocreators-ee/apity/dist/fetcher.js b/node_modules/@cocreators-ee/apity/dist/fetcher.js
index 8399670..157a22c 100644
--- a/node_modules/@cocreators-ee/apity/dist/fetcher.js
+++ b/node_modules/@cocreators-ee/apity/dist/fetcher.js
@@ -37,11 +37,50 @@ function getQuery(method, payload, query) {
             delete payload[key];
         });
     }
+    // support filters and populations from strapi endpoints
+    else if (payload.filters || payload.populate) {
+        let { filters = {}, populate = [],  ...rest } = payload;
+        queryObj = {  ...rest };
+        
+        Object.entries(flattenObj(filters)).forEach(([key, value]) => {
+            queryObj[convertDotsToBrackets(`filters.${key}`)] = value
+        })
+
+        // see https://docs.strapi.io/dev-docs/api/rest/guides/understanding-populate#populate-components
+        if (typeof populate === 'string') {
+            if (populate.includes(',')) {
+                populate = populate.split(',').map((value) => value.trim());
+            } else {
+                populate = [populate];
+            }
+        }
+        populate.forEach((value, index) => {
+            queryObj[`populate[${index}]`] = value;
+        });
+    }
     else {
         queryObj = { ...payload };
     }
     return queryString(queryObj);
 }
+
+function isPlainObj(o) {
+    return o && o.constructor && o.constructor.prototype && o.constructor.prototype.hasOwnProperty("isPrototypeOf")
+}
+
+function flattenObj(obj, keys=[]) {
+    return Object.keys(obj).reduce((acc, key) => {
+        return Object.assign(acc, isPlainObj(obj[key]) 
+            ? flattenObj(obj[key], keys.concat(key))
+            : {[keys.concat(key).join(".")]: obj[key]}
+        )
+    }, {})
+}
+
+function convertDotsToBrackets(str) {
+    return str.replace(/\.(.+?)(?=\.|$)/g, (m, s) => `[${s}]`)
+}
+
 function getHeaders(body, init) {
     const headers = new Headers(init);
     if (body !== undefined && !headers.has('Content-Type')) {
@@ -83,7 +122,7 @@ export function getFetchParams(request) {
     return { url, init };
 }
 async function getResponseData(response) {
-    const contentType = response.headers.get('content-type');
+    const contentType = response.headers.get('content-type')?.split(';')?.[0]; // remove the encoding information;
     if (response.status === 204) {
         return undefined;
     }
